pyhealth/datasets/base_dataset.py

```diff
@@ -468,26 +472,29 @@ def load_table(self, table_name: str) -> dd.DataFrame:
                     operator.add, (df[col].astype(str) for col in timestamp_col)
                 )
             else:
-                # Single timestamp column
-                timestamp_series: dd.Series = df[timestamp_col].astype(str)
+                # Single timestamp column - don't convert to string yet
+                timestamp_series: dd.Series = df[timestamp_col]
```

Comment:
This would work for the `else` case, though i think the if case is also problematic is some of the columns are `NA`. But we can worry about it in a later PR.